### PR TITLE
Columbus interface

### PR DESCRIPTION
**Pull request by SoubhikM**

### CODE REVIEW

**Review by gayverjr — COMMENTED** [review]
Couple of comments:
- Please review the [numpy](https://numpydoc.readthedocs.io/en/latest/format.html) style doc strings, specifically for parameters and returns. It's important to have the type in there for documentation.  So for example, in `dm_from_iwfmt` the returns section should read something like:

```
Returns 
-----------
dm1_mo: np.ndarray
    1 particle density matrix in MO basis
````
- The DRT number is currently hard coded as 1. This can easily be relaxed by simply adding DRT1 and DRT2 as keyword arguments which are both set to 1 by default in `tdm_ao` and `sdm_ao`. 
- Just a matter of style, i would prefer the default for the `correction` keyword argument in `colparser.H0` to be set to 'pople' rather than None, because having it be None and then setting it to not None is kinda misleading
- On a similar note, no need to use *args in `colparser.H0_parser` when we only use the first one. Just make it a positional argument.
- I will ask you to drop the `__main__` function later but its fine to keep it there for now for testing.

File: pyopencap/analysis/ColumbusParser.py [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+                self.nmpsy = ([int(x) for x in line.split()[2:len(line)]])
+                line=next(rfile)
+                self.nfcpsy = ([int(x) for x in line.split()[2:len(line)]])
+                break
+
+        self.mos = self.read_molden(molden_file)
+        self.set_mo_coeff()
+
+
+    def irrep_info(self):
```
Comment (gayverjr) [inline pyopencap/analysis/ColumbusParser.py:158]: Good idea creating this function, just needs proper numpy docstring

**Review by gayverjr — APPROVED** [review]
Looks great! I'll add my N2- example and write a quick set of tests.

**Comment by SoubhikM** [comment]
I have a test script ready, if you want, I can push that too.

**Comment by gayverjr** [comment]
Go ahead and make a new pull request for it!

**Comment by gayverjr** [comment]
I only request that you make the pull request on the 'dev' branch rather than main.

**Pull request merged**